### PR TITLE
Add ollydbg.scyllahide.vm package

### LINKED ISSUE #164: Package proposal: ollydbg.scyllahide.vm
### Package Name

ollydbg.scyllahide

### Tool Name

ollydbg.scyllahide

### Package type

OTHER/UNKNOWN

### Tool's version number

2021-08-23_13-27-50 

### Category

Debuggers

### Tool's authors

x64dbg

### Tool's description

Anti-anti-debugging plugin for Ollydbg

### Download URL

https://github.com/x64dbg/ScyllaHide/releases/download/snapshot-2021-08-23_13-27-50/ScyllaHide.7z

### Download SHA256 Hash

c51929341ff726d219e670928433a176e114ca9a4c36f416629aef50c98b8817

### Why is this tool a good addition?

It provides anti-anti-debugging for debuggers

### PR DESCRIPTION
**Pull request by TheoKlein**
Add ollydbg.scyllahide.vm package to close #164

### CODE REVIEW

**Comment by mr-tz** [comment]
Thank you! Could you update this like @MalwareMechanic did in #227?

**Comment by TheoKlein** [comment]
Sure, it's much better!

File: packages/ollydbg.scyllahide.vm/tools/chocolateyinstall.ps1 [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+    VM-Remove-PreviousZipPackage ${Env:chocolateyPackageFolder}
+
+    $toolSrcDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+    $toolSrcDir = Join-Path $toolSrcDir 'ScyllaHide'
+
+    $packageArgs = @{
+    packageName   = ${Env:ChocolateyPackageName}
```
Comment (mr-tz) [inline packages/ollydbg.scyllahide.vm/tools/chocolateyinstall.ps1:11]:
```suggestion
        packageName   = ${Env:ChocolateyPackageName}
```

File: packages/ollydbg.scyllahide.vm/tools/chocolateyinstall.ps1 [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        checksum      = 'c51929341ff726d219e670928433a176e114ca9a4c36f416629aef50c98b8817'
+        checksumType  = 'sha256'
+    }
+
+    Install-ChocolateyZipPackage @packageArgs
+    VM-Assert-Path $toolSrcDir
+
+    $toolDstDir = Join-Path ${Env:RAW_TOOLS_DIR} 'OllyDbg' -Resolve
+    if (-Not (Test-Path $toolDstDir -PathType Container)) {
+        New-Item -ItemType directory $toolDstDir -Force -ea 0
```
Comment (mr-tz) [inline packages/ollydbg.scyllahide.vm/tools/chocolateyinstall.ps1:23]:
```suggestion
        New-Item -ItemType directory $toolDstDir -Force -ea 0 | Out-Null
```

File: packages/ollydbg.scyllahide.vm/tools/chocolateyinstall.ps1 [hunk 3]
```diff
@@ -0,0 +1,32 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+    VM-Remove-PreviousZipPackage ${Env:chocolateyPackageFolder}
```
Comment (mr-tz) [inline packages/ollydbg.scyllahide.vm/tools/chocolateyinstall.ps1:5]: @MalwareMechanic should this be removed here as well?

**Review by mr-tz — CHANGES_REQUESTED** [review]
thanks, just a few minor things I've noticed before we can merge

**Review by mr-tz — APPROVED** [review]
great, thank you!

**Pull request merged**